integrate/yigconf/yig.toml

```diff
@@ -11,6 +11,7 @@ ssl_key_path = ""
 ssl_cert_path = ""
 
 # DebugMode
+lcdebug = true
```

Comment:
This config should be set as flase by default.

Reply:
The config that you specified has been deleted as same as before.

Reply:
The config you specified has been deleted and remains the same as before.